models/User.js

```diff
@@ -0,0 +1,31 @@
+const Schema = require ('mongoose')
+
+
+
+const itemSchema = new Schema({
```

Comment:
Maybe rename to Item from itemSchema because you create it with "new Schema" so it's obvious that it's Schema and please notice about name convention for Models (from uppercase letter)